website/src/pages/tabs.mdx

```diff
@@ -91,6 +91,7 @@ Please see the [styling guide](/styling).
 | [onChange](#tabs-onchange)         | function      | false    |
 | [defaultIndex](#tabs-defaultindex) | number        | false    |
 | [index](#tabs-index)               | number        | false    |
+| [orientation](#tabs-orientation)   | string        | false    |
```

Comment:
Is `string` enough for the type here? Or should I make it more specific, a la [line 250](https://github.com/reach/reach-ui/pull/486/files#diff-2db3c8b9bcec3e34f466572fcfd6df2bR250)?